docker/tests/Dockerfile

```diff
@@ -0,0 +1,153 @@
+# Multi-stage build
+# Use: docker build --no-cache --build-arg PPA_TRACK="[staging|stable] GOVERSION=[1.20.6|1.18|?]"
+
+# Build 0 - fraken
+FROM golang:alpine AS fraken-builder
+RUN apk add --no-cache -t .build-deps \
+  autoconf \
+  automake \
+  bison \
+  build-base \
+  curl \
+  file \
+  file-dev \
+  flex \
+  git \
+  jansson \
+  jansson-dev \
+  jansson-static \
+  libc-dev \
+  libmagic \
+  libmagic-static \
+  libtool \
+  linux-headers \
+  openssl \
+  openssl-dev \
+  openssl-libs-static \
+  py3-setuptools \
+  python3 \
+  python3-dev \
+  sudo
+
+RUN set -x \
+  && echo "Compiling Yara from source..."
+
+# Fetch and compile libyara
```

Comment:
this and the actual fraken build can be replaced by:
COPY --from=us-docker.pkg.dev/osdfir-registry/turbinia/release/fraken:latest` /bin/fraken /opt/fraken/fraken

when https://github.com/google/turbinia/pull/1353 is in

Reply:
done